examples/django/proj/celery.py

```diff
@@ -7,14 +7,16 @@
 # set the default Django settings module for the 'celery' program.
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'proj.settings')
 
+from django.conf import settings # noqa
+
 app = Celery('proj')
 
 # Using a string here means the worker will not have to
 # pickle the object when using Windows.
-app.config_from_object('django.conf:settings', namespace='CELERY_')
+app.config_from_object('django.conf:settings')
```

Comment:
Namespace here should be CELERY_ to add a prefix to all Celery settings (new in 4.0)